book_source/adv_user_guide_web/using_browndog.Rmd

```diff
@@ -1 +1,11 @@
-# Using Browndog
\ No newline at end of file
+# Using Browndog
+
+The Browndog service provides PEcAn with access to large and diverse sets of data at the click of a button in the format that PEcAn needs.
+
+## Current Capabilities 
+
+The current integration of Browndog into PEcAn allows you to access the NARR, CRUNCEP, and AmerifluxLBL Meteorological data products iwthout using your local machine to download and process the raw data.
+
+** More Informatoin can be found at the Browndog website**
```

Comment:
Typos in this sentence and the one before it. Also, I'd just link to the met product page rather than listing the specific products, it will make it easier to keep this page up-to-date if it doesn't require updating when products are added